services/web-app/src/app/components/header/header.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 import Image from "next/image";
-import { SignInButton } from "@/components/buttons/signIn";
+import { SignInButton } from "@/app/components/buttons/signIn";
```

Comment:
Components should be outside the app dir. Same for lib. 

App is just for pages and layouts etc. 
Components is for reusable tsx pieces which will be rendered.
Lib is for reusable functions and states etc.